src/components/scene/editor.js

```diff
@@ -0,0 +1,40 @@
+/* global AFRAME */
+var registerComponent = require('../../core/component').registerComponent;
+
+var EDITOR_URL = 'https://aframe.io/aframe-editor/build/aframe-editor.js';
```

Comment:
We should version this later `https://aframe.io/aframe-editor/builds/0.3.0/aframe-editor.js`